conda/petsc/build.sh

```diff
@@ -16,7 +16,7 @@ export CXXFLAGS=$(echo ${CXXFLAGS:-} | sed -E 's@\-fdebug\-prefix\-map[^ ]*@@g')
 export FFLAGS=$(echo ${FFLAGS:-} | sed -E 's@\-fdebug\-prefix\-map[^ ]*@@g')
 export FCFLAGS="$FFLAGS"
 export HYDRA_LAUNCHER=fork
-export LIBS="-lmpifort -lgfortran"
+#export LIBS="-lmpifort -lgfortran"
```

Comment:
```suggestion
```

Reply:
Beat me to pressing the button! I agree.